src/Entity/QuickNodeCloneEntityFormBuilder.php

```diff
@@ -67,4 +140,36 @@ public function getForm(EntityInterface $original_entity, $operation = 'default'
     return $this->formBuilder->buildForm($form_object, $form_state);
   }
 
+  /**
+   * Check whether to exclude the paragraph field.
+   *
+   * @param $pfield_name
+   *
+   * @return bool
+   */
+  public function excludeParagraphField($pfield_name) {
```

Comment:
Passing in the bundle here might simplify this check as you can check the config using bundle as key

Reply:
The way the config has been set up you can't check the config using the bundle (unless I've misunderstood your point above), which in this case is 'paragraph'. It currently checks it using the paragraph bundle e.g. 'group' to get the fields. I've also tried checking the config using the field name. 

I suspect that this will be possible when all the config is nested as in the generic solution, but this implementation is too prescriptive.

Reply:
It's currently checking the field against every available paragraph bundle, but should only check against the bundle of the current paragraph. We could pass in `$value->entity->bundle()` to solve this.

Reply:
Resolved, turned out I had misunderstood. Talking in real life helped resolve the issue.